projects/angular/src/layout/vertical-nav/_vertical-nav.clarity.scss

```
@@ -271,6 +267,8 @@
     .nav-header {
       //Dims
       padding: vertical-nav-variables.$clr-vertical-nav-header-padding;
+      height: vertical-nav-variables.$clr-vertical-nav-item-height;
+      @include mixins.overflow-ellipsis();
```

Will be ebetter the have `min-height` and not overflow elipsed.
 
```suggestion
      min-height: vertical-nav-variables.$clr-vertical-nav-item-height;
```